util/test/sub_test

```diff
@@ -1262,6 +1268,7 @@ for testname in testsrc:
         # remember to add the cleaner solution soon.
         #
         comptimeout = 2*timeout
+        cmd=ShellEscapeCommand(cmd);
```

Comment:
Is this mainly to deal with the task/thread model macros that turn into #includes (e.g. `-DCHPL_TASKS_MODEL_H="tasks-qthreads.h"`)? I think @gbtitus has talked about getting rid of those in the past by using a common name for the headers.

I think this is useful regardless, just curious if that issue was still present/motivating this change.